idea_town/frontend/static-src/app/templates/experiment-page.js

```diff
@@ -6,9 +6,9 @@ export default `
         <div class="details-header">
           <h1 data-hook="title"></h1>
           <div class="idea-controls">
-            <button data-hook="feedback" class="button primary">Give Feedback</button>
-            <button data-hook="uninstall" class="button neutral">Disable <span data-hook="title"></span></button>
-            <button data-hook="install" class="button primary">Enable <span data-hook="title"></span></button>
+            <button data-hook="feedback" id="feedback-button" class="button primary">Give Feedback</button>
+            <button data-hook="uninstall" id="uninstall-button" class="button neutral"><span class="state-change-inner"></span><span class="transition-text">Disabling...</span><span class="default-text">Disable <span data-hook="title"></span><span></button>
```

Comment:
This... doesn't look kosher.

There is something funny around the end bit of `</span><span></button>`.
Should be `</span></span></button>` since we're in a nested `span`.